src/nrniv/nrnbbcore_write.cpp

```diff
@@ -1902,7 +1902,7 @@ bool is_coreneuron_loaded() {
     // check if corenrn_version symbol can be found
```

Comment:
Comment needs to be changed.
Note: is it really the case that corenrn_version exists but not always corenrn_embedded_run? How can that be?

Reply:
Ohh. Is it because of the case of static linking?

Reply:
Here is my experience so far:

* we have two libraries : libcoreneuron (contains corenrn_version)  and libcorenrnmech (contains corenrn_embedded_run) 
* we dlopen libcorenrnmech (which also brings symbol of libcoreneuron because it's linked)
* this issue doesn't happen on OS X
* if we don't close the handle to libcorenrnmech then everything works fine.

So my guess (without reading anything) is when `dlopen("libcorenrnmech")` somehow leaves `libcoreneuron` symbol in memory space.

Reply:
Ok. I was mistaken in thinking that corenrn_embedded_run would be in libcoreneuron.